docs/src/openapi-v1.yaml

```diff
@@ -685,6 +685,12 @@ paths:
           description: If true, does not return metadata hex. This is useful when
             metadata is not needed and response time is a concern. Defaults to false.
           default: false
+      - name: decodeMeta
+        in: query
+        schema:
+          type: boolean
+          description: If true, the metadata returned will be decoded and JSON formatted. 
+          default: false
```

Comment:
I think this needs removing again, then? :)

Reply:
Yea the docs still need to get updated :)